src/algorithm/destination.rs

```diff
@@ -0,0 +1,62 @@
+use num_traits::{Float, FromPrimitive};
+use types::Point;
+
+/// Returns a new Point using the distance to the existing Point and a bearing for the direction
+
+pub trait Destination<T: Float> {
+    /// Returns a new Point using distance to the existing Point and a bearing for the direction
+    ///
+    /// ```
+    /// # extern crate geo;
+    /// # #[macro_use] extern crate approx;
+    /// #
```

Comment:
you shouldn't need these three lines for this doc example. `extern crate geo` is already implicitly added and you aren't using the 'approx' provided `assert_relative_eq!` macro